### ISSUE TITLE
Extension was removed from both stores, try to fix violations

### ISSUE THREAD

**Issue by RunninglVlan** [RunninglVlan/Facebook-Status#8]
[AppDetex](https://www.appdetex.com/), who reported violations to stores, referred to [Brand Guidelines for Facebook Developers](https://developers.facebook.com/docs/app-review/resources/logos/) explaining why my extension was removed.
Try to fix violations - change "Facebook" text (keep it only in description), replace icons (@iconmonstr, @thenounproject), etc.
Change name to Social Network Status?
Maybe add disclaimer in description, like in [Notifier for Facebook](https://chrome.google.com/webstore/detail/notifier-for-facebook/ocgmcodaanlenepilchadlnabakhgaoi/related)

**Comment by Aleck85** [comment]
Still not fixed? Perhaps you eventually make Firefox version instead of unworking chrome/opera?

**Comment by RunninglVlan** [comment]
Didn't have time (probably motivation) yet. I can try to look into Firefox version in the next few days.

Fixed in #9.

**Comment by ghost** [comment]
will the opera version be back up?  tried workplace status but that won't work for me with facebook.

**Comment by RunninglVlan** [comment]
Have you somehow lost the last version from Opera store? I'm still using that one. =)
Have you tried installing this extension from local machine through _Developer mode_?
I will try to do something about this task.

**Comment by ghost** [comment]
nah, got the last version from opera installed.. and now it's working again  o_O
no, i didn't install fb status through developer mode, installed it through opera store months ago. 
i shall be very happy about it working though :D
i tried the workstatus extension, for some reason that won't work, will see if that issue remains on my side, but i am glad fb status works again.
thank you so much for both of these superb extensions though, much much appreciated  ^_^
